### PR TITLE
Fix typo in parameter type

### PR DESCRIPTION
**Pull request by bbbbx**
as title

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @bbbbx!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/CesiumGS/cesium/blob/main/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.
* :grey_question: Unit tests were not updated.
    * Make sure you've [updated tests](https://github.com/CesiumGS/cesium/tree/main/Documentation/Contributors/TestingGuide) to reflect your changes, added tests for any new code, and ran the code coverage tool.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.

**Comment by ggetz** [comment]
Thanks for the fix @bbbbx!

File: Source/Core/Matrix4.js [hunk 1]
```diff
@@ -1675,7 +1675,7 @@ const scaleScratch5 = new Cartesian3();
  * Extracts the rotation matrix assuming the matrix is an affine transformation.
  *
  * @param {Matrix4} matrix The matrix.
- * @param {Matrix4} result The object onto which to store the result.
+ * @param {Matrix3} result The object onto which to store the result.
  * @returns {Matrix4} The modified result parameter.
```
Comment (j9liu) [inline Source/Core/Matrix4.js:1679]: Sorry that this comment is late, but it looks like the `@returns` should also be a `Matrix3` as well?

Comment (ggetz) [reply Source/Core/Matrix4.js:1679]: Tweaked in https://github.com/CesiumGS/cesium/commit/77d458504b8e4d22cf860a21610b144b5e4b0e6b.

**Pull request merged**